src/Symfony/Component/Process/Tests/ProcessTest.php

```diff
@@ -1166,6 +1166,24 @@ public function pipesCodeProvider()
     }
 
     /**
+     * @dataProvider provideVariousIncrementals
+     */
+    public function testIncrementalOutputDoesNotRequiresAnotherCall($stream, $method) {
+        $process = new Process("php -r '\$n = 0; while (\$n < 3) { fwrite($stream, \$n, 1); \$n++; usleep(1000); }'", null, null, null, null);
+        $process->start();
+        usleep(50000);
```

Comment:
are you sure this won't create a transient test? (I didn't check extensively, just asking)